java/java-ranger-regression/printtokens_eqchk/impl/printTokens2.java

```diff
@@ -20,7 +20,6 @@ public static void main(String[] args) {
 	}
 	
 	public static void mainProcess(char i0, char i1, char i2, char i3, char i4, char i5, char i6, char i7){
-		output = 0;
 		
 //		StringBuilder sb = new StringBuilder();
```

Comment:
It would be good to add a comment why this code is commented out. Otherwise, just remove the commented code.

Reply:
I've left a comment explaining this commented code. The commented code is actually part of the original benchmark which reflects changes made by the authors who distributed these benchmarks.